packages/block-library/src/template-part/index.php

```diff
@@ -15,8 +15,8 @@
 function render_block_core_template_part( $attributes ) {
 	$content = null;
 
-	if ( ! empty( $attributes['postId'] ) ) {
-		// If we have a post ID, which means this template part
+	if ( ! empty( $attributes['postId'] ) && is_string( get_post_status( $attributes['postId'] ) ) ) {
```

Comment:
Do we need to also check if is a `string`? If the post doesn't exist it will return `false` and if exists it will return a `truthy` string value. https://developer.wordpress.org/reference/functions/get_post_status/
```suggestion
	if ( ! empty( $attributes['postId'] ) && get_post_status( $attributes['postId'] ) ) {
```

Reply:
Makes sense... Pushed suggested fix :+1: